.devcontainer/compose.yaml

```diff
@@ -0,0 +1,50 @@
+services:
+  rails:
+    build:
+      context: ..
+      dockerfile: .devcontainer/Dockerfile
+
+    volumes:
+      - ../..:/workspaces:cached
+
+    # Overrides default command so things don't shut down after the process ends.
+    command: sleep infinity
+
+    depends_on:
+      - postgres
+      - redis
+      # - memcached
+      - selenium
+
+    # Use "forwardPorts" in **devcontainer.json** to forward an app port locally.
+    # (Adding the "ports" property to this file will not forward from a Codespace.)
+    ports:
+      - 45678:45678
```

Comment:
I'd propose adding an env variable to improve the corepack prompt on the install process

```
environment:
      COREPACK_ENABLE_DOWNLOAD_PROMPT: 0 # prevent yarn install y/n prompt
```